src/components/Navigation.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import homeIcon from './../img/groceries.svg';
+import './../styles/Navigation.css';
 
-const Navigation = () => {
+const Navigation = ({ confirmDeleteUserToken }) => {
```

Comment:
This incorporates changes made by @rachel-fischoff to the Navigation, but integrates them with my changes to include the SVG.